stable/kubecost/values.yaml

```diff
@@ -0,0 +1,192 @@
+# Default values for kubecost parent chart (wraps kubecost cost-analyzer chart).
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+hooks:
+  # Modifies the prometheus configmap to set the prometheus cluster_id
+  # external label to the cluster's kube-system ns uid.
+  # Creates configmap to pass kube-system ns uid as envvar to kubecost.
+  clusterID:
+    enabled: true
+    kubectlImage: "bitnami/kubectl:1.16.2"
+
+cost-analyzer:
+  enabled: true
+
+  global:
+    prometheus:
+      # If false, Prometheus will not be installed -- only actively supported on paid Kubecost plans
+      enabled: true
+
+    thanos:
+      enabled: false
+
+    grafana:
+      # If false, Grafana will not be installed
+      enabled: true
+
+  ingress:
+    enabled: true
+    annotations:
```

Comment:
I think we should remove these annotations. Specially for the chart, this may be in part the issue we are seeing, it also is too specific and tied to our own use case.

Reply:
ok! I will move them to comments for users to override in values if they want this to work with a konvoy traefik setup

Reply:
I am still looking into this, I thought ingress enabled should work anyway... checking a few things.